src/transaction_builder.js

```diff
@@ -71,16 +71,14 @@ export class TransactionBuilder {
     if (!sourceAccount) {
       throw new Error('must specify source account for the transaction');
     }
-    this.source = sourceAccount;
-    this.operations = [];
 
     if (isUndefined(opts.fee)) {
-      // eslint-disable-next-line no-console
-      console.log(
-        '[TransactionBuilder] The `fee` parameter of `TransactionBuilder` is required. Future versions of this library will error if not provided.'
-      );
+      throw new Error('must specify fee for the transaction');
```

Comment:
Should this error include an example of the expected value? E.g. `throw new Error('must specify fee for the transaction in stroops (e.g. StellarSdk.BASE_FEE or 100)`.

Reply:
That's a good idea. I'm hesitant to call out the library name because it's technically StellarBase.BASE_FEE, but I'm guessing the vast majority of people would see it in StellarSdk.BASE_FEE, and I don't want to invite confusion from either side.